src/py42/services/orgs.py

```diff
@@ -182,6 +214,8 @@ def get_agent_full_disk_access_states(self, org_id):
         """
         return self.get_agent_state(org_id, "fullDiskAccess")
 
+    # TODO - status of this org settings with api clients
```

Comment:
The is an important `TODO`.  With the latest apis, a lot of the old org settings code doesn't work well with artemis only tenants, because they lack the expected backup settings. 

We should decide how we want to handle `get_settings` and `update_settings` for api-clients. I would favor on the side of marking them as deprecating them/marking them incompatible with api-clients, especially since I'm not sure its even possible to make them fully backwards compatible - but I'm open to suggestions. We could certainly make them _as close to backwards compatible as possible_, but the new endpoint for updating orgs doesn't include any backup configuration.

Reply:
I'm fine with just leaving this as a TODO for now. Before we do a release with api client support, we'll need to come up with a strategy for disabling/raising on things that don't/won't support them (like this and the web/push restore methods). But we don't quite need to come up with that strategy _now_.